src/components/involve/involve.js

```diff
@@ -43,7 +52,69 @@ const GetInvolve = props => {
             </div>
             <div className="involve-image"></div>
         </div>
-    </>;
+        
+        <LargeTextBlock 
+            text={text}
+        />
+        <LargeTextBlock 
+            title="Title"
+            text={text}
+        />
+        <LargeTextBlock 
+            text={prompt}
+            expand={text2}
+            clickable={true}
+        />
+        <LargeTextBlock 
+            title="Title"
+            text={prompt}
+            expand={text2}
+            clickable={true}
+        />
+        <LargeTextBlock 
+            text={prompt}
+            expand={text2}
+            clickable={true}
+            hidePrev={true}
+        />
+        <LargeTextBlock 
+            title="Title"
+            text={prompt}
+            expand={text2}
+            clickable={true}
+            hidePrev={true}
+        />
+
+        <TextBlockButton 
+            text={prompt}
+            expand={text2}
+        />
+        <TextBlockButton 
+            title="Title"
+            text={prompt}
+            expand={text2}
+        />
+
+        <ProjectBlock
+            name="Project Name"
+            year="2020"
+            professor="XXXX XXXX"
+            img="logo512.png"
+            description={project}
+            details={project2}
+            award="Gold"
+        />
+        <ProjectBlock
+            name="Project Name"
+            year="2019"
+            professor="XXXX XXXX"
+            img="logo512.png"
+            description={project}
+            details={project2}
+            award="Silver"
+        />
```

Comment:
You can programmatically create this instead of hardcoding it, using an array and `Array.map`.